client/signup/steps/p2-site/index.jsx

```diff
@@ -439,7 +588,26 @@ class P2Site extends Component {
 			return this.props.translate( 'Site created - Go to next step' );
 		}
 
-		return this.props.translate( 'Continue' );
+		return this.props.translate( 'Create workspace' );
+	};
+
+	renderFormNotice = () => {
+		return (
+			<div className="p2-site__workspace-form-notice-wrapper">
+				<hr />
+				<div className="p2-site__workspace-form-notice">
+					<div className="p2-site__workspace-form-notice-icon">
+						<Gridicon size={ 24 } icon="info-outline" />
+					</div>
+					<p>
+						{ this.props.translate(
+							"The first P2 in your workspace will be created automatically for you, so you can focus on getting started. You'll be able to customize it at any time!"
+						) }
+					</p>
+				</div>
+				<hr />
+			</div>
+		);
 	};
 
 	renderSubdomainSuggestions() {
```

Comment:
Do you think we should drop this functionality as it sounds very similar to the new one and might be confusing for devs? Or perhaps give them some kind of different naming.

Also, not sure about the look once you get to this state:
![Screenshot on 2022-03-17 at 12-01-48](https://user-images.githubusercontent.com/4988512/158795500-eccc3d7d-4147-4091-be33-c3bf6d6c12cc.png)

Reply:
Yes, personally... I'd remove it completely, I think the new automatic suggestions in the input itself should be enough, if the custom domain input is shown and an already existing domain exists probably just showing the red error message and letting the user click the "choose a suggestion" link would be enough probably?
 
@david-gonzalez-a8c what do you think?

Reply:
I agree. I also think it's also what's described in the designs, as I don't see this state anywhere.

Reply:
Great, I've just removed it.

**note**: When entering a custom domain site, the validation is still being performed only when clicking the "Create workspace" button, same as it was before. I think a nice improvement would be adding some real time validation as you type the custom url, but I think we can do this later as a separate PR to reduce the bloat in this one.